### ISSUE TITLE
Creating title and description using a template parameter

### ISSUE THREAD

**Issue by ggppdk** [FLEXIcontent/flexicontent-cck#502]
Title and description

would be created via a template parameter that would support replacements:
- language strings
- field values and field labels
- item properties (e.g. item id)
- other ?

The above will happen on item save, but
- we will also need a progressive AJAX task, to update existing items, similar to the AJAX tasks for file sizing in files manager and in search indexer

Also about the we already have a function that does the above replacements:
```
static function replaceFieldValue( &$field, &$item, $variable, $varname, & $cacheable = false )
```
we can duplicate the code and may small adjustments

**Comment by micker** [comment]
yes to override title of column in defaut template it will be great

**Comment by ggppdk** [comment]
no no that is not it,
i meant add automatic data
- to db table  "content"
- to the search index

thus you can **hide item title and item description in item form**
and they will be created automatically

**Comment by micker** [comment]
ok i understand
